packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -383,6 +389,7 @@ export class GraphQLApi extends Construct {
           )
           : undefined,
       additionalAuthenticationProviders: this.formatAdditionalAuthenticationProviders(props),
+      xrayEnabled: props.xrayEnabled || false,
```

Comment:
The property doesn't need to be configured here. If it is undefined in CloudFormations, Cfn will configure the AppSync without x-ray support by default.

```suggestion
      xrayEnabled: props.xrayEnabled,
```

Reply:
@BryanPan342 
Thank you for your comment.
I will fix it along with the integration test.